File: aula-07/vetor/java/src/test/java/com/github/jc00973/ufg/cs/aula07/LocalizarPalavraFrequenteTest.java
package com.github.jc00973.ufg.cs.aula07;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertThrows;
import static org.junit.Assert.*;

public class LocalizarPalavraFrequenteTest {

    @Test
    void testLocalizarPalavraFrequente() {
        assertEquals("Joao", AlgoritmosVetor.localizarPalavraFrequente("Joao Jose Joao Joao Joao Jose"));
        assertEquals("Sexta", AlgoritmosVetor.localizarPalavraFrequente("Quarta Sexta Quinta Sexta Sexta Quarta"));
    }

}

File: aula-07/vetor/js/test/localizarPalavraFrequente.test.js
const funcoes = require("../AlgoritmosVetor");

test("teste 1", () => {
    expect(funcoes.localizarPalavraFrequente("Joao Jose Joao Joao Joao Jose Marilda")).toBe("Joao");
    expect(funcoes.localizarPalavraFrequente("Terca Quarta Sexta Quinta Sexta Sexta Quarta")).toBe("Sexta");
});

